src/frontend/src/hooks/work-packages.hooks.ts

```diff
@@ -44,10 +45,16 @@ export const useSingleWorkPackage = (wbsNum: WbsNumber) => {
  * @param wpPayload Payload containing all information needed to create a work package.
  */
 export const useCreateSingleWorkPackage = () => {
-  return useMutation<{ message: string }, Error, any>(['work packages', 'create'], async (wpPayload: any) => {
-    const { data } = await createSingleWorkPackage(wpPayload);
-    return data;
-  });
+  return useMutation<{ message: string }, Error>(
+    ['work packages', 'create'],
+    async (wpPayload: CreateWorkPackageApiInputs | void) => {
+      if (!wpPayload) {
```

Comment:
this is the wrong way to fix this. Remove the if check and the | void, and add CreateWorkPackageApiInputs as the third input to the useMutation generic: `useMutation<{message: string}, Error, CreateWorkPackageApiInputs>`. These inputs in the generic are telling the useMutation what to expect for a return, an error, and an input